apps/webpack.js

```diff
@@ -143,40 +143,25 @@ if (envConstants.HOT) {
 
 // modify baseConfig's preLoaders if looking for code coverage info
 if (envConstants.COVERAGE) {
-  baseConfig.module.rules.splice(
-    -1,
-    1,
-    {
-      test: /\.jsx?$/,
```

Comment:
The previous config boldly assumed that the last entry in baseConfig.module.rules was responsible for loading .js and .jsx files, and then overwrite it to exclude `apps/src` so that `babel-istanbul-loader` could load and instrument it instead.

The new `istanbul-instrumenter-loader` doesn't require us to omit `apps/src` from the initial `babel-loader` step and instead adds instrumentation as a `post` step.